htdocs/public/payment/paymentok.php

```diff
@@ -394,7 +394,7 @@
 
 			// Do action only if $FinalPaymentAmt is set (session variable is cleaned after this page to avoid duplicate actions when page is POST a second time)
 			if (!empty($FinalPaymentAmt) && $paymentTypeId > 0) {
-				$result = $object->validate($user);
+				$result = $object->status == -2 ? -1 : $object->validate($user); // if membre is excluded (status == -2) the new validation is not possible
```

Comment:
Can you replace the -2 with $object::STATUS_RESILIATE ?

Reply:
Only the excluded member should not be able to reopen their account with public payment. 

Why a resiliated member couldn't change his mind and reactivate his account ?

Reply:
Your commit is much better thx